src/forms/renderers/OAuth.tsx

```diff
@@ -115,37 +123,69 @@ const OAuthproviderRenderer = ({
     };
 
     return (
-        <>
-            <Typography>
-                <FormattedMessage
-                    id="oauth.instructions"
-                    values={{ provider: capitalizedProvider }}
-                />
-            </Typography>
-
-            {hasLength(errorMessage) ? (
-                <Alert severity="error">{errorMessage}</Alert>
-            ) : null}
-
-            <Stack direction="row" spacing={2}>
-                <Alert severity="warning">
-                    Under Development - do not use in prod
-                </Alert>
-
-                {provider === 'google' ? (
-                    <GoogleButton disabled={loading} onClick={openPopUp} />
-                ) : null}
+        <Box
+            sx={{
+                mt: 2,
+            }}
+        >
+            {loading ? <FullPageSpinner /> : null}
 
-                {isAuthorized ? (
-                    <>
-                        <Alert severity="success">Authenticated</Alert>
-                        <Button onClick={removeCofig}>Remove</Button>
-                    </>
+            <Stack direction="column" spacing={2}>
+                <Typography>
+                    <FormattedMessage
+                        id="oauth.instructions"
+                        values={{ provider }}
+                    />
+                </Typography>
+
+                {hasLength(errorMessage) ? (
+                    <Alert
+                        severity="error"
+                        sx={{
+                            maxWidth: '50%',
+                        }}
+                    >
+                        {errorMessage}
+                    </Alert>
                 ) : null}
-            </Stack>
 
-            {loading ? <FullPageSpinner /> : null}
-        </>
+                <Stack
+                    direction="row"
+                    spacing={2}
+                    sx={{
+                        alignItems: 'center',
+                    }}
+                >
+                    {providerVal === 'google' ? (
+                        <GoogleButton disabled={loading} onClick={openPopUp} />
+                    ) : (
+                        <Button disabled={loading} onClick={openPopUp}>
+                            <FormattedMessage
+                                id="oauth.authenticate"
+                                values={{ provider }}
+                            />
+                        </Button>
+                    )}
```

Comment:
Right now we only specifically handle Google. Otherwise we display a normal button.